tokenizers/src/processors/bert.rs

```diff
@@ -105,6 +115,9 @@ impl PostProcessor for BertProcessing {
                 [&vec![0u32; encoding.get_type_ids().len()][..], &[1]].concat();
             let pair_attention_mask = vec![1; pair_ids.len()];
 
+            // For compatibility with `TemplateProcessing`, the sequence_ranges shouldn't contain
+            // the special tokens.
+            let pair_sequence_ranges = HashMap::from_iter(vec![(1, 0..pair_ids.len() - 1)]);
```

Comment:
It's not exactly clear why this code needs to care about the ranges and those hardcoded values feel a bit scary (-1, +1 etc..).

Could be at least have a test that would exhibit the error in case this line was wrong ? It would be even better if we didn't need the hardcodedness.